ncow-api/src/main/scala/net/ncow/ncow/api/NcowService.scala

```diff
@@ -1,34 +1,33 @@
 package net.ncow.ncow.api
 
 import akka.{Done, NotUsed}
-import com.lightbend.lagom.scaladsl.api.broker.Topic
-import com.lightbend.lagom.scaladsl.api.broker.kafka.{KafkaProperties, PartitionKeyStrategy}
+import com.lightbend.lagom.scaladsl.api.transport.Method
 import com.lightbend.lagom.scaladsl.api.{Service, ServiceCall}
 import play.api.libs.json.{Format, Json}
-import com.lightbend.lagom.scaladsl.api.transport.Method
 
 trait NcowService extends Service {
 
-  def registerBot(botRegistrationRequest: BotRegistrationRequest): ServiceCall[BotRegistrationRequest, BotRegistrationResponse]
+  def registerBot: ServiceCall[RegisterBotRequest, RegisterBotResponse]
 
   override final def descriptor = {
     import Service._
+    // @formatter:off
```

Comment:
Do we know what these do?

Reply:
I think it prevents the editor from screwing up with the indentation.